lib/proteus/kit.rb

```diff
@@ -16,7 +16,13 @@ def new(kit_name, repo_name = nil)
 
       if system "git ls-remote #{url(kit_name)} #{repo_name} > /dev/null 2>&1"
         puts "Starting a new proteus-#{kit_name} project in #{repo_name}"
-        system "git clone #{url(kit_name)}#{' ' + repo_name}"
+        system "git clone #{url(kit_name)}#{' ' + repo_name} && \
+          cd #{repo_name} && \
```

Comment:
if the repo name has strange characters, it will fail in odd ways. How about putting double quotes around it, and using `%{....}` to quote the whole big string being passed to `system`?